app/controllers/api/trainees/withdraw_controller.rb

```diff
@@ -2,7 +2,7 @@
 
 module Api
   module Trainees
-    class WithdrawsController < Api::BaseController
+    class WithdrawController < Api::BaseController
```

Comment:
This should really be named `WithdrawalController` I think

Reply:
Then routes is going to be like 

```ruby
resource :withdrawal, only: %i[create], path: "/withdraw"

```
vs
```ruby
resource :withdraw, only: %i[create]
```

:shrug: too much withdrawing symptoms :stuck_out_tongue_winking_eye: